### PR TITLE
pints 1.1.13

### PR DESCRIPTION
**Pull request by edmundmiller**
The bioconda builds aren't getting updated, and the biocontainers
containers aren't getting updated. They're publishing to PyPI
consistently, so we'll meet them where they're at.

Using Seqera Containers because this isn't possible with biocontainers that I know of to use pypi packages.

https://github.com/nf-core/modules/issues/5814 is the issue with the linting.

### CODE REVIEW

**Comment by maxulysse** [comment]
issue with lint

File: modules/nf-core/pints/caller/main.nf [hunk 1]
```diff
@@ -5,8 +5,8 @@ process PINTS_CALLER {
     conda "${moduleDir}/environment.yml"
     // NOTE Stopped publishing at 1.1.9 https://quay.io/repository/biocontainers/pypints?tab=tags
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'oras://community.wave.seqera.io/library/pints_caller:650ef63c9768bb5a' :
-        'community.wave.seqera.io/library/pints_caller:b1e6404513917871' }"
+        'oras://community.wave.seqera.io/library/pybedtools_bedtools_htslib_pip_pypints:aa20de1f1b5ddb30' :
```
[the remark this answers is not in the document]
Comment (edmundmiller) [reply modules/nf-core/pints/caller/main.nf:8]: Ooo, didn't know that came out!

[the remark this answers is not in the document]
Comment (edmundmiller) [reply modules/nf-core/pints/caller/main.nf:8]: How do you get one if you use wave-cli? I don't think I can use a environment.yml on the site...

[the remark this answers is not in the document]
Comment (edmundmiller) [reply modules/nf-core/pints/caller/main.nf]: Figured it out

**Pull request merged**